Wiki-Labels-Guide.md

```diff
@@ -0,0 +1,83 @@
+# Wiki Labels Guide
+Labels help in categorizing [**issues**](https://github.com/FreeCodeCamp/wiki/issues) and [**pull requests**](https://github.com/FreeCodeCamp/wiki/pulls). Use **_issue_ only** labels for issues and **_PR_ only** labels for pull requests. Listed here are labels for [**FreeCodeCamp Wiki**](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki) &mdash; see [*wiki source*](https://github.com/FreeCodeCamp/wiki) and [*wiki labels*](https://github.com/FreeCodeCamp/wiki/labels).
+
+Labels | Description | Use in Issue | Use in PR
+:---|:---:|:---:|:---:
+[**![Label: QA](https://img.shields.io/badge/Label-QA-0052cc.svg?style=flat-square&maxAge=2592000 "Label: QA")**](https://github.com/FreeCodeCamp/wiki/labels/QA) | Content Review :book: | :x: | :white_check_mark: 
+[**![Label: help wanted](https://img.shields.io/badge/Label-help_wanted-159818.svg?style=flat-square&maxAge=2592000 "Label: help wanted")**](https://github.com/FreeCodeCamp/wiki/labels/help wanted) | Need help :heavy_plus_sign: in creating the article(s) | :white_check_mark: | :x: 
+[**![Label: content update](https://img.shields.io/badge/Label-content_update-eb6420.svg?style=flat-square&maxAge=2592000 "Label: content update")**](https://github.com/FreeCodeCamp/wiki/labels/content update) | Update :up: non-code contents of article(s) | :x: | :white_check_mark:
+[**![Label: enhancement](https://img.shields.io/badge/Label-enhancement-30A9DE.svg?style=flat-square&maxAge=2592000 "Label: enhancement")**](https://github.com/FreeCodeCamp/wiki/labels/enhancement) | Enhance codes or algos :memo: in article(s) | :x: | :white_check_mark:
+[**![Label: WORK NEEDED:](https://img.shields.io/badge/Label-WORK_NEEDED:-f9a11b.svg?style=flat-square&maxAge=2592000 "Label: WORK NEEDED:")**](https://github.com/FreeCodeCamp/wiki/labels) | Work needed :construction_worker: on article(s) | :x: | :white_check_mark:
```

Comment:
@SaintPeter Just to be consistent and show that it's not a label in itself rather it's a prefix to a bunch of other labels. Same for `LANG:` as this is not a label in itself rather it's namespace for a bunch of other language specific labels. (although language specific labels are not name spaced)